osu.Game/Screens/Multi/Match/MatchSubScreen.cs

```diff
@@ -195,8 +195,13 @@ private void currentItemChanged(ValueChangedEvent<PlaylistItem> e)
 
             Beatmap.Value = beatmapManager.GetWorkingBeatmap(localBeatmap);
             Mods.Value = e.NewValue?.RequiredMods?.ToArray() ?? Array.Empty<Mod>();
+
             if (e.NewValue?.Ruleset != null)
+            {
+                Ruleset.Disabled = false;
```

Comment:
Will be fixed via https://github.com/ppy/osu-framework/pull/2570, these changes should be removed.